plugins/ufm_log_analyzer_plugin/src/loganalyze/log_analyzer.py

```diff
@@ -369,7 +369,10 @@ def create_analyzer(
         end = time.perf_counter()
         log.LOGGER.debug(f"Took {end-start:.3f} to load the parsed data")
 
-        all_images_outputs_and_title = ufm_top_analyzer.full_analysis()
+        all_images_outputs_and_title, dataframes_for_pdf, lists_to_add = (

```

Comment:
A general comment about the PR description:
How - I don't think we should have a link to Redmine here. This is a public repo, and this link does not help anyone from outside trying to understand things. 
Test- Please give more information on how it was tested

Reply:
Test section was changed

Reply:
and the why?

Reply:
I think it's useful and convenient that the link is there.

Reply:
Please remove it....
1. Not everyone who has access to the code has access to the ticket.
2. The PR itself should be clear in a way that there is no need to go to other links.
3. The ticket might be changed/deleted - what you write here will stay with the code.

If we move to another ticket system in two years and all the old tickets are lost, a reader will not have the full context of the code.

Reply:
I removed it